snippets/fsharp.json

```diff
@@ -0,0 +1,184 @@
+{
+    "Lambda function": {
+        "prefix": "fx",
+        "body": [
+            "(fun ${1:x} -> $2) $0"
+        ],
+        "description": "Lamba function"
+    },
+	
+	// Surround width snippets
```

Comment:
JSON doesn't support comments, I put them in there anyway to emphasize the groups, I'll remove them once and if want to commit the snippets